frontends/mit-open/src/page-components/TestimonialDisplay/TestimonialDisplay.tsx

```diff
@@ -24,6 +42,7 @@ const QuoteContainer = styled.section(({ theme }) => ({
   marginBottom: "80px",
   [theme.breakpoints.down("md")]: {
     marginBottom: "40px",
+    height: "200px",
```

Comment:
Do we want to fix the height for mobile? I'd avoid scroll inside scroll where possible.

<img width="570" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/b7292053-c86e-4e68-b820-77e929882937">

Reply:
Should be fixed in https://github.com/mitodl/mit-open/pull/1131/commits/062a703348e430bcce1c389a15a821e064f74fe0 - think I nerfed this accidentally during a recent rebase.